docs/topics/modules/viewing_build_pipeline_logs.adoc

```diff
@@ -0,0 +1,14 @@
+[id="viewing_build_pipeline_logs"]
+= (Optional) Viewing the build pipeline logs
```

Comment:
This has changed to optional but it's not meant to be. While the user waits, we definitely want them to go see the logs to get an idea of what's happening (and see how to access this information in their OSO account)

Reply:
Viewing the build pipeline logs is an optional section even in the original doc. I distinctly remember back in the demo days this was used to divert attention from the long wait time for a pipeline to go from one stage to another. I have tested this around 3 or 4 times now and the pipelines are executed fairly faster now. 
This section is purely for troubleshooting (for advanced users) as the first paragraph clearly points out and imho we could safely add this to the User guide/ a detailed use case for pipelines section. This is definitely not something a user has to or should do as part of promoting his application. As to 'letting the user know what's happening' the UX has become fairly descriptive now, actually telling users what is happening, especially when the pipeline is first triggered. I actually liked that!